content/attacks/posts/2023-09-15-Remitano.md

```diff
@@ -0,0 +1,50 @@
+---
+date: 2023-09-14
+target-entities: Remitano
+entity-types:
+  - Exchange
+  - Custodian
+attack-types:
+  - Wallet Hack
+title: "Remitano Crypto Exchange Suffers $2.7 Million Loss in Exploit"
+loss: 2700000
+---
+
+## Summary
+
+On September 14, 2023, Remitano, a cryptocurrency exchange, fell victim to a security breach, resulting in unauthorized transactions on the Ethereum and TRON blockchains and a significant financial loss of $2.7 million. This incident was primarily a hack of the exchange's hot wallet, triggered by a data leak from a third-party source. Tether's prompt intervention helped freeze the attacker's addresses, securing 1.9 million USDT and averting further potential losses. The stolen assets were converted to 163 ETH and transferred to HitBTC.
+
+## Attackers
+
+The identity of the attacker is unknown. The following addresses are associated with this attack:
+
+   - [0x74530e81e9f4715c720b6b237f682cd0e298b66c](https://etherscan.io/address/0x74530e81e9f4715c720b6b237f682cd0e298b66c)
+   - [TEDNf1aqk8YJEUdNH9NRd4MqibZmdP49Fm](https://tronscan.org/#/address/TEDNf1aqk8YJEUdNH9NRd4MqibZmdP49Fm)
+
+## Losses
+
+Remitano lost approximately $2,700,000 during the attack and $1.9 million was frozen by Tezer. The drainer swapped the stolen USDC and ANKR for 163 ETH (264000 USD) and [transferred them to HitBTC](https://twitter.com/PeckShieldAlert/status/1702507801669775712).
+
+Assets stolen on ETH:
+
+- [1,359,253 USDT](https://etherscan.io/tx/0xe0725362fd774de0d8416d5e3d028063508ffa61f68087c576320e42159677a9)
+- [208,188 USDC](https://etherscan.io/tx/0x574a416cf868e3439da278ae7043c7eb364a76d323dddf7c5b1c155628843c5f)
+- [34.4 ETH](https://etherscan.io/tx/0xef9a766c6a7a935ee26f4d1e36cd7c19a14b84559c2d350be3aca217273c7891)
+- [104,360 ANKR](https://etherscan.io/tx/0x5fb5119be8b588edf198fd2de73ba8e5a3758474176be7e721f5cb841009f532)
+
+Assets stolen on Tron:
+
+- [537,915 USDT](https://tronscan.org/#/address/TEDNf1aqk8YJEUdNH9NRd4MqibZmdP49Fm/transfers)
```

Comment:
The link refers to an address, not a transaction. Replace with a transaction link as in the link below.